show/platform.py

```diff
@@ -5,20 +5,40 @@
 import click
 import utilities_common.cli as clicommon
 from sonic_py_common import device_info, multi_asic
+from swsscommon import swsscommon
 
 
+#
+# Constants
+# 
+
+CHASSIS_INFO_TABLE = 'CHASSIS_INFO|chassis {}'
+CHASSIS_INFO_CARD_NUM_FIELD = 'module_num'
+CHASSIS_INFO_SERIAL_FIELD = 'serial'
+CHASSIS_INFO_MODEL_FIELD = 'model'
+CHASSIS_INFO_REV_FIELD = 'revision'
+
+# Get hardware information
 def get_hw_info_dict():
```

Comment:
can you check whether we should move this function to https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py?

Reply:
I've commented about this function before. 

> The device_info file is frankly a hodge-podge of various methods of obtaining information from the device… we have calls to STATE_DB and CONFIG_DB as well as direct file parsing, and command dispatching. It also handles the return of many different types of information including the base mac address, the routing stack, file paths to various config files, data from other config files. Due to this, I do have some reservations modifying this file as it is not very maintainable in my opinion. It may be our best option though. 

I don't know if I am in a position to make that determination, there doesn't seem to be any firm rules about what goes into that file, its a "smorgasbord" of platform related functions. 

If you or someone else with experience thinks it is best I am glad to do it.

Reply:
I don't have a strong opinion on this, but also I don't see anything wrong to move this function to device_info.py.